features-sdk/step_definitions/global.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Nitpicking: "global" steps related to the API are in the `controller-steps.js` file. Maybe it's not explicit enough and you can rename it but I would like to avoid having too many files

Reply:
oh, I didn't found them... I'll move this step in the right file right now.